cmd/cluster.go

```diff
@@ -32,6 +32,18 @@ func setupClusterCommand() *cobra.Command {
 		},
 	}
 
+	pullCommand := &cobra.Command{
+		Use:   "pull",
+		Short: "Pulls down the most recent version of the images",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := cluster.DockerComposePull()
```

Comment:
I prefer not to leak the Docker internals from the package (package). How about `cluster.Update()`?